neuraxle/pipeline.py

```diff
@@ -354,9 +354,9 @@ class MiniBatchSequentialPipeline(NonFittableMixin, Pipeline):
     Mini Batch Sequential Pipeline class to create a pipeline processing data inputs in batch.
     """
 
-    def __init__(self, steps: NamedTupleList, batch_size):
+    def __init__(self, steps: NamedTupleList):
         Pipeline.__init__(self, steps)
-        self.batch_size = batch_size
+        self.is_initialized = True
```

Comment:
Why this? I wouldn't thing it is already initialized. A pipeline should be initialized only when `setup` has been called on each of the steps it contains. Why would the pipeline be already ready?

Reply:
added missing .setup() calls in fit, and fit_transform for the MiniBatchSequentialPipeline